test/connect.test.js

```diff
@@ -5,30 +5,36 @@
  */
 
 /*
- * Copyright (c) 2014, Joyent, Inc.
+ * Copyright 2020 Joyent, Inc.
  */
 
-var Logger = require('bunyan');
+var assert = require('assert-plus');
+var bunyan = require('bunyan');
 var clone = require('clone');
 
 var UFDS = require('../lib/index');
 
 
 // --- Globals
 
-var UFDS_URL = 'ldaps://' + (process.env.UFDS_IP || '10.99.99.18');
+assert.string(process.env.UFDS_IP, 'UFDS_IP envvar');
+assert.string(process.env.UFDS_LDAP_ROOT_PASSWORD,
+    'UFDS_LDAP_ROOT_PASSWORD envvar');
 
-var LOG = new Logger({
+var UFDS_URL = 'ldaps://' + process.env.UFDS_IP;
+var UFDS_PASSWORD = process.env.UFDS_LDAP_ROOT_PASSWORD
```

Comment:
This is missing a semicolon ... which then breaks `make check`.

Reply:
Sorry about that. See https://github.com/joyent/node-ufds/pull/6